buildSrc/src/main/kotlin/ManifestPatcher.kt

```diff
@@ -0,0 +1,31 @@
+import java.io.File
+
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+private fun rewriteCrateVersion(line: String, version: String): String = line.replace(
```

Comment:
since this stuff is in buildSrc now I wonder if we should add some unit tests...

Reply:
Good call!